commands/uploader.go

```diff
@@ -332,13 +332,16 @@ func (c *uploadContext) Await() {
 }
 
 var (
+	githubHttps, _ = url.Parse("https://github.com")
+	githubSsh, _   = url.Parse("ssh://github.com")
```

Comment:
If you add a `/` at the end of these, `strings.HasPrefix(u.Path, supported.Path)` will protect against urls like `https://github.com:123/...`.

Also, does LFS add the `ssh://` prefix for git remotes like `git@github.com:...`?